### ISSUE TITLE
Add health_check to nginx

### ISSUE THREAD

**Issue by nixelsolutions** [ehazlett/interlock#128]
Hey @ehazlett many thanks for this wonderful project! I'm wondering if you could add health_check on upstream configuration, as sometimes some containers keep failing with 40x or 50x error codes and those are noted by users as interlock is not dropping those failed containers from the LB pool.

It would be ideal if we could parametrize the URI, attempts, and so on :-)

Thanks a lot!

**Comment by ehazlett** [comment]
You should be able to do it now with the `interlock.health_check` label.  Try that and see if that works.

**Comment by chromerobv** [comment]
looking thru code looks like interlock.health_check is only for haproxy.
the only way i think to do this would be to use an nginx image that is built with nginx-upstream_check and then add code the interlock to use the variables and add some code to the template. i've done this for a standalone nginx-proxy if you'd like me add it to interlock.

**Comment by ehazlett** [comment]
Yes we could extend the label to work for nginx.  A contribution would be much apprecitated! :)

**Comment by chromerobv** [comment]
https://github.com/ehazlett/interlock/pull/179
